src/components/widgets/Number.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import Header from './Header'
+import "../../styles/widget/number.css"
+import "../../styles/widget/widget.css"
+
+class Number extends Component {
+
+  calculateProgress( mainStat, goal ) {
```

Comment:
`mainStat` and `goal` are available via `this.props` - we should not be passing this data around since the function already has access to props here (this does not leverage encapsulation and creates an unnecessarily complex interface).

Also, please use complete words for variable names, and omit the types from the variable name.  

Finally, there's a surprising amount of calculation condensed into this one function.  Consider breaking up into functions that each have their own responsibility:
```jsx
progress() {
  const { statistic, goal } = this.props

  return ( statistic / goal * 100 ).toFixed( 2 )
}
```
Consider using string interpolation instead of `progressNum.toString().concat('%')` - it feels overly complicated when we could use:
```jsx
`${this.progress()}%`
```